cardigan/stories/data/links.ts

```diff
@@ -0,0 +1,27 @@
+export const links = [
```

Comment:
I think this could live within the story if it's just in there, I believe this folder's used for shared date (like using the same images everywhere).